flutter/test/sentry_native_channel_test.dart

```diff
@@ -6,239 +6,228 @@ library flutter_test;
 import 'package:flutter_test/flutter_test.dart';
 import 'package:mockito/mockito.dart';
 import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_flutter/src/native/factory.dart';
 import 'package:sentry_flutter/src/native/method_channel_helper.dart';
-import 'package:sentry_flutter/src/native/sentry_native.dart';
-import 'package:sentry_flutter/src/native/sentry_native_channel.dart';
+import 'package:sentry_flutter/src/native/sentry_native_binding.dart';
+import 'mocks.dart';
 import 'mocks.mocks.dart';
+import 'sentry_flutter_test.dart';
 
 void main() {
-  group('$SentryNative', () {
-    late Fixture fixture;
+  for (var platform in [
+    MockPlatform.android(),
+    MockPlatform.iOs(),
+    MockPlatform.macOs()
+  ]) {
```

Comment:
this is basically the only addition here - running the test for all implementations of SentryNativeBinding